README.md

```diff
@@ -1,5 +1,12 @@
-# extension-lti-deliveryprovider
-The LTI Delivery extension allows test-takers to take a delivery (delivered test) via LTI
+# TAO _ltiDeliveryProvider_ extension
+
+![TAO Logo](https://github.com/oat-sa/taohub-developer-guide/raw/master/resources/tao-logo.png)
+
+![GitHub](https://img.shields.io/github/license/oat-sa/extension-tao-ltideliveryprovider.svg)
+![GitHub release](https://img.shields.io/github/release/oat-sa/extension-tao-ltideliveryprovider.svg)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/y/oat-sa/extension-tao-ltideliveryprovider.svg)
+
+> The LTI Delivery extension allows test-takers to take a delivery (delivered test) via LTI

```

Comment:
Why you format it that way?

Reply:
Because that's copied from the template: https://github.com/oat-sa/readme-templates/tree/master/extension-core/extension-tao-ltideliveryprovider